applications/dashboard/scss/editor/_richEditor.scss

```diff
@@ -676,3 +677,65 @@ $embedded-quote_fg: #A3A3A3 !default;
     color: inherit;
     font-weight: $richEditor-atMentionLink_weight;
 }
+
+// Handle gutter for paragraph menu
+
+
+
+body.hasRichEditor {
+    // New Discussions
+    &.Section-PostDiscussion {
+        #DiscussionForm {
+            select {
+                max-width: 100%;
+            }
+
+            .FormWrapper {
+                padding: {
+                    left: $richEditor-pilcrow_gutter;
+                }
+
+                .richEditor {
+                    padding-left: $richEditor-pilcrow_gutter;
```

Comment:
Shouldn't this be removed? It's still padded inwards even with the extra gutter on the side.

![image](https://user-images.githubusercontent.com/1770056/39062511-c5b3e6c2-4495-11e8-8cce-2522c7791ae1.png)